userChrome.css

```diff
@@ -9,14 +9,14 @@
   Description:    Minimalist, Simple, Keyboard Centered and based on SimpleFox. 🦊
                   What you get is a really simple responsive one-line layout using the new Proton UI.
 
-                  > SimpleFox: https://github.com/migueravila/SimpleFox
+                  Based on SimpleFox: https://github.com/migueravila/SimpleFox
 
 
   Author:         Andreas Grafen
                   (https://andreas.grafen.info)
 
 
-  Repository:     https://github.com/andreasgrafen/ag.proton
+  Repository:     https://github.com/andreasgrafen/cascade
```

Comment:
This one is good tho. I didn't even realise it's still the old link. >_>

Reply:
i don't think this is a change, so ill resolve it

Reply:
Yeh, not a change. I just like to misuse features.. :D